src/connections/DelayedJob.ts

```diff
@@ -0,0 +1,57 @@
+import { createSelector } from "reselect";
+
+import { DelayedJobsFindVariables, listDelayedJobs } from "@/generated/v3/jobService/jobServiceComponents";
+import { delayedJobsFindFetchFailed } from "@/generated/v3/jobService/jobServicePredicates";
+import { DelayedJobDto } from "@/generated/v3/jobService/jobServiceSchemas";
+import { ApiDataStore } from "@/store/apiSlice";
+import { Connection } from "@/types/connection";
+import { connectionHook } from "@/utils/connectionShortcuts";
+import { selectorCache } from "@/utils/selectorCache";
+
+type DelayedJobsConnection = {
+  delayedJobs?: DelayedJobDto[];
+  isLoading: boolean;
+  hasLoadFailed: boolean;
+};
+
+type DelayedJobsProps = { uuid: string };
+
+const delayedJobsSelector = (store: ApiDataStore) => {
+  const delayedJobsMap = store.delayedJobs || {};
+  return Object.values(delayedJobsMap).map(resource => resource.attributes);
+};
+
+const delayedJobsLoadFailedSelector = (store: ApiDataStore, { uuid }: DelayedJobsProps) => {
+  const variables: DelayedJobsFindVariables = {
+    pathParams: { uuid }
+  };
```

Comment:
Since the API request doesn't use these path params, the load failed selector shouldn't either. The params to those two must always match for this feature to function.

Reply:
understood and removed